commands/commands.js

```diff
@@ -0,0 +1,26 @@
+const ImagesService = require("../services/Services");
+const dayjs = require('dayjs')
+
+const imagesService = new ImagesService();
+
+const helloCommand = (msg) => {
+    msg.reply(`Hello ${msg.author.username}, have a good day!`);
+};
+
+const memeCommand = async(msg) => {

```

Comment:
All right, just here you should add an option in case there's an error and the boot doesn't break

Reply:
Thank you very much for the observation, this change was made, you can see it here: 50852bb04f617eee39fa92b201b92662dcda15ee